### PR TITLE
New package: thunar-archive-plugin-0.3.1

### PR DESCRIPTION
**Pull request by LockeAnarchist**

### CODE REVIEW

File: srcpkgs/thunar-archive-plugin/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'thunar-archive-plugin'
+pkgname=thunar-archive-plugin
+version=0.3.1
+revision=1
+build_style=gnu-configure
+configure_args="--prefix=/usr --sysconfdir=/etc --libexecdir=/usr/lib/xfce4 --localstatedir=/var --disable-static"
```
Comment (xtraeme) [inline srcpkgs/thunar-archive-plugin/template]: This line is redundant and wrong, remove it.

File: srcpkgs/thunar-archive-plugin/template [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+makedepends="Thunar-devel exo-devel"
+hostmakedepends="intltool pkg-config xfce4-dev-tools"
+depends="Thunar"
```
Comment (xtraeme) [inline srcpkgs/thunar-archive-plugin/template]: This is also unnecessary and redundant.

File: srcpkgs/thunar-archive-plugin/template (hunk 1 above)
Comment (LockeAnarchist) [inline srcpkgs/thunar-archive-plugin/template]: at least libexecdir is necessary, I think

Comment (xtraeme) [inline srcpkgs/thunar-archive-plugin/template]: No, --libexecdir is completely wrong. We *do* libexec unlike Arch.

**Comment by silvernode** [comment]
no wonder I couldn't extract or create archives in Thunar. Thanks for adding this!

**Pull request merged**